plugins/UM3NetworkPrinting/src/UltimakerNetworkedPrinterOutputDevice.py

```diff
@@ -293,8 +295,16 @@ def _updatePrinters(self, remote_printers: List[ClusterPrinterStatus]) -> None:
 
     def _checkIfClusterHost(self):
         """Check is this device is a cluster host and takes the needed actions when it is not."""
-
         if len(self._printers) < 1 and self.isConnected():
+            self._num_is_host_check_failed += 1
+        else:
+            self._num_is_host_check_failed = 0
+
+        # Since we request the status of the cluster itself way less frequent in the cloud, it can happen that a cloud
+        # printer reports having 0 printers (since they are offline!) but we haven't asked if the entire cluster is
+        # offline. (See CURA-7360)
+        # So by just counting a number of subsequent times that this has happened fixes the incorrect display.
+        if self._num_is_host_check_failed >= 6:
```

Comment:
Where does the number 6 comes from?

Reply:
IIUC from the difference in the intervals between the checks. One check is done every 10 seconds and the other every 2 seconds. That would allow up to 5 failed checks in between the 10 second interval. Perhaps we should calculate this from the constants it's derived from.